composer_class.rb

```diff
@@ -0,0 +1,21 @@
+class Composer
+  attr_accessor :first_name, :last_name, :year_born, :year_died, :age
+  @@all = []
+  @@selected = []
```

Comment:
This class variable will always be the same as `@@all`, because there is no functionality to set it, add or remove composers to it. It will always include all the composers that have been created.

I suggest some code examples below, but check it for mistakes, they are just guesses and I haven‘t tried them.